cmd/internal/apispec/flags.go

```diff
@@ -62,17 +62,23 @@ func init() {
 		"out",
 		"The location to store the spec. Can be an AkitaURI or a local file. Defaults to a new spec on Akita Cloud.")
 
+	Cmd.Flags().StringVar(
+		&serviceFlag,
+		"project",
+		"",
+		"Your Akita project. Exactly one of --out, --cluster, or --project must be specified.")
```

Comment:
The old description of `--service` suggests that both `--out` and `--service` were required if `--out` didn't specify an AkitaURI.